src/pkg/lint/lint_test.go

```diff
@@ -93,3 +95,67 @@ func TestFillObjTemplate(t *testing.T) {
 	require.ElementsMatch(t, expectedFindings, findings)
 	require.Equal(t, expectedComponent, component)
 }
+
+func TestLintPackageWithImports(t *testing.T) {
```

Comment:
In an ideal world we would testing more cases which are smaller. But this makes sense as an iterative step in the right direction.